packages/preview/tinyset/0.1.0/lib.typ

```diff
@@ -0,0 +1,31 @@
+/// You must create a header to get the rest of the style
+#let header(number: 1, class: "Math 3551 - Fall 2024", name: "Your Name") = {
+  set text(font: "New Computer Modern Math")
+  align(center)[
+    #class \
+    #name - Homework #number
+  ]
+  set par(justify: true)
```

Comment:
I don't think that will work as expected, since `set` rules are scoped. This would only apply until the end of the current scope, which is on the next line. If you want to apply this rule to the whole document, the correct way to do it is to use a global `show` rule. There is [a tutorial on how to create one](https://typst.app/docs/tutorial/making-a-template/).

Reply:
Yes of course! Thanks for pointing that out, I will move the show rules into the individual components, just for ease of use of the library.